includes/abstracts/abstract-wc-product.php

```diff
@@ -1809,7 +1809,7 @@ public function get_image( $size = 'woocommerce_thumbnail', $attr = array(), $pl
 			$image = '';
 		}
 
-		return apply_filters( 'woocommerce_product_get_image', wc_get_relative_url( $image ), $this, $size, $attr, $placeholder, $image );
+		return apply_filters( 'woocommerce_product_get_image', $image, $this, $size, $attr, $placeholder );
```

Comment:
We still need the last `$image` here, as removing that is a breaking change.

Reply:
Yup, so this has to wait for 4.0, if I get the SemVer right, correct? 
How do we flag such things for update when we can introduce breaking change (so that we won't forget to remove duplicate parameter)?

Reply:
Yep you got it; we can't make breaking changes until 4.0. 👍 You can put a [todo](https://github.com/woocommerce/woocommerce/blob/094fc1bf3a25bd5e42becabbb219a7d9f52077ea/includes/admin/settings/class-wc-settings-checkout.php#L6) to flag it for removal in the next major release.